### ISSUE TITLE
No antemeridian support?

### ISSUE THREAD

**Issue by StyXman** [GIScience/openrouteservice#1278]
#### Here's what I did

Tried to reproduce this route on your site.

#### Here's what I got

To begin with, the web interface seems to fail to even be able to display both ends. At some point it zoomed all the way out to 0 and showed the start point in the East end of the map and the finish point on the West end of the map. Routing also seems to fail.

#### Here's what I was expecting

just drive a couple of KM over the antemeridian.

**Comment by koebi** [comment]
Hey,

your issue seems to be missing a link to the route you were trying to reproduce.
Could you also confirm that "your site" is maps.openrouteservice.org?

Thanks :)

**Comment by sfendrich** [comment]
This might be a data issue. E.g. the nodes https://www.openstreetmap.org/node/3732714612 and https://www.openstreetmap.org/node/3732825786 are placed at 180 and -180 degrees and do not connect the two road segments west and east of the antimeridian.

**Comment by StyXman** [comment]
> your issue seems to be missing a link to the route you were trying to reproduce

oups, sorry. here it is:

https://www.openstreetmap.org/directions?engine=fossgis_osrm_car&route=68.91640%2C-179.79540%3B68.96970%2C-180.11400

and here's the link I got on your site that I got after trying to et up a similar route but on another road:

https://maps.openrouteservice.org/#/directions/Iul'tinskiy,CK,Russia/Iul'tinskiy,CK,Russia/data/55,130,32,198,15,97,4,224,38,9,96,59,2,24,5,192,166,6,113,0,184,64,70,1,216,4,224,14,152,227,240,5,159,0,217,136,25,159,124,5,96,9,129,182,1,165,182,211,104,108,80,149,54,68,0,50,136,104,69,128,110,0,180,68,203,16,1,198,210,177,122,107,27,81,231,192,70,225,248,197,82,162,195,72,46,32,32,0,117,79,2,34,108,57,64,2,242,128,22,215,43,75,215,160,64,6,111,0,3,110,139,142,0,9,230,4,20,128,14,103,45,14,131,16,10,228,28,141,1,98,11,238,143,238,128,152,134,10,23,128,153,14,238,238,136,139,14,139,2,0,11,227,84,0

it also fails. but if it's a matter of bad data, we can drop this one.
